Lib/socket.py

```diff
@@ -463,6 +465,34 @@ def fromfd(fd, family, type, proto=0):
     nfd = dup(fd)
     return socket(family, type, proto, nfd)
 
+if hasattr(_socket.socket, "sendmsg"):
+    def send_fds(sock, msg, fds):
```

Comment:
Please make the function signature similar to the function signature of ``sendmsg``:
```
send_fds(sock, buffers, fds[, flags[, address]])
```